VisualPinball.Unity/Documentation~/creators-guide/setup/installing-vpe.md

```diff
@@ -13,12 +13,9 @@ In order to start creating or modifying tables with VPE, the first thing you'll
 
 Unity uses an application called *Unity Hub* to update itself, create new projects and provide quick access to them. The install process is straight-forward and documented [here](https://docs.unity3d.com/Manual/GettingStartedInstallingHub.html) if you run into troubles.
 
-When installing Unity, use the latest **2021.2** version. You can leave all the other options unchecked.
+When installing Unity, use the latest **2021.3.0f1** version. You can leave all the other options unchecked.

```

Comment:
Not sure I would say `latest 2021.3.0f1` since there is really just one `2021.3.0f1`.  

Since the hub always tries to fetch the latest, (as of today, `2021.3.8f1`), the user has to install from the archives. 

In the Volley readme here is the line:

``` 
Note: Anything above Unity 2021.3.0 currently doesn't work. In order to install 2021.3.0, open the Unity hub, choose *Installs*, click *Install Editor*, *Archive*, and on the *download archive* link. Then, under *Unity 2021.x*, click the *Unity Hub* button under *Unity 2021.3.0f1 (12 Apr, 2022)*. This should get to you back into the hub, where you can install 2021.3.0.
```